### DISCUSSION TITLE
Thoughts on Indian Diplomas, Degrees and all that.

### DISCUSSION THREAD

**Discussion by Vyvy-vi** [EddieHubCommunity/support#2478] [General]
The educational scenario is a slippery slope and no one can really comment with 100% certainty if your path is right or not.
:) Believe in the path you choose and the path you want to go, continue to work towards that, pivot when needed... It all makes sense somewhere down the line. 

A Diploma in Cs and a degree(B.Tech/B.Sc./BCA) are the two entry-points into academic Computer Science over here.
@krishdevdb queried about what would be a good option for them to pursue... 11th-12th-degree or a diploma, followed by a degree?.

**This is my opinion** (feel free to give insights in the comments on this discussion. If you disagree, feel free to contextualise it, and mention what I might be getting wrong here!).
Sadly, here in India, people tend to discriminate against people that have a diploma only(now yes, I agree that diploma doesn't cover all of the B.Tech syllabi... but at the same time it does cover the technical topics in detail and is a much more practical approach. Now, 11th and 12th do have importance, but depending on your career choices, a large share of the syllabus would be redundant to you. That, plus how the current educational flow results in the "learn a lot before the exam, dump it out in the exam and then that's the sliding point and most people don't retain a large share of the information" + the pressure of competitive exams isn't very healthy).
However, IMHO going the 11th-12th grade way is a good option too.

## The Pros of going the 11th-12th-degree way-
- Math Syllabus (It may not seem like it, however, the Math syllabus of class 11th and 12th is an introduction to what you would most probably need in university(a CS degree is more than just programming), and would help you when you're working on specific problems...)
- More career options, opportunities. Sadly, a large share of companies directly disqualify people that haven't finished school.
- More options. Say you did a diploma in CS, and after that, you got excited about another parallel field like Applied Math or something? It would be harder for you to switch to the field.
- Organised syllabi. I think college tends to be slightly relieving in the first year or so, which gives you the time to figure things out, try things and participate in things, which help you move closer to what specifically do you want. (*Idk how rigorous diploma courses are)
- Networking benefits. You could network on your own, however a university that has 2000 students or more access you something unique- You can converse, connect and network with people that are potentially going to be doing great things moving forward, and you could help them or build things up with them. (*Idk how many people are there in a diploma course, but my guess is that it's not a lot of people)
- More education opportunities. A large share of colleges well-known for computer science don't have provisions for admissions for people that did a diploma. (*there are a few, but I think those turn into lateral entry examinations... which is essentially held for both diploma students and people in their 2nd year of college).
- Foreign unis/exchange programs most probably need you to have cleared school.

## The Cons of going the 11th-12th degree way-
- A large share of the syllabus(esp. chemistry) tends to not have particular relevance to your future career routes, and that kinda leads to not paying enough attention to these, which might lead to worse performance.
- 11th and 12th is hectic, and can get very stressful for a few people(esp. when you're preparing for multiple competitive examinations, to get into a good uni... The culture tends to become slightly toxic too... I remember missing classes on a few days due to health reasons, and people refused to share their notes... 😦 )
- Getting at the top unis, is relatively harder for general category candidates(*I don't mean to bring up the debate about reservation, let's not comment on that.. Regardless, the above is true), and often it isn't related to how good you are at programming.
- Confusion, exploitation and scams: A large share of 'institutions' have a habit of confusing individuals a bit and exploiting your parent's money... One scam that I'm not fond of(and was affected by) is - "Integrated Programs" that a few schools run. Coaching is also a complicated topic, and many times even though individuals pay relevant amounts, and perform considerably okay, there's a segregation of children into batches... which is a mess. Your school hypes up Board exams, and in the process, they make your pre-boards ultra-hard sometimes(and grade them mercilessly), which results in worse mid-term cards.. and that's something that could affect you if you're also pursuing foreign unis.

Personally, my grade 11 was very messy... I worked really really hard, but things just didn't seem to click. Exams just didn't end(every week, there was something you need to prepare for), and for some reason, my school signed me up for all the olympiad examinations(which was messed up, I tried reading engineering books or higher-level books, which made me confused and very burnt out)

## The Bottomline-
I think y'all can make it in either way, as long as you consistently work on things that accelerate you on the paths you chose.
You could balance the above pros and cons, and maybe make a choice. One piece of advice that I would give is- "Be sure that the things you're doing make sense to you, and aren't a result of peer pressure. Try being nice to people, if you're in a competitive atmosphere... and no matter if you feel like you have fallen behind in the race, try and look to the next thing instead of contemplating or blaming the system's issues or sit with your regrets. Allow yourself to have the time you need to develop yourself and have the courage to seek knowledge that can empower you. It's not a race, you'll get there eventually... just don't lose yourself on the way!"

**Comment by shreemaan-abhishek** [comment]
Wonderful!
